spec/services/course/survey/survey_download_spec.rb

```diff
@@ -49,7 +49,7 @@
       context 'header' do
         it 'fifth element onwards is question descriptions in increasing weight' do
```

Comment:
Should be `sixth element onwards`. Or can be even more explicit with `sixth element (inclusive) onwards`.